pages/users/user_delete.php

```diff
@@ -0,0 +1,28 @@
+<?php
+restrict_access(Access::$EDIT_USERS);
+
+$user_id = get_route_param("user_id");
+$user = em()->find(USER::class, $user_id);
+if (!$user) {
+    echo "the user of id $user_id doesn't exist";
+    return;
+}
+if (!empty($_POST) and isset($_POST['delete'])) {
+    em()->remove($user);
```

Comment:
Soft delete here. Hard delete will either be very radical, or not work at all. With soft delete we can have another error message on login (`your account has be deactivated` or something)

Reply:
Maybe what we could do if:
- soft delete with the action button in `user_list`
- hard delete only for root or in dev environment AND if the user has no FK data (otherwise FK constraint will block deletion unless `cascade: remove` is on the association - see doctrine docs)